Jenkinsfile.conda

```diff
@@ -0,0 +1,72 @@
+pipeline {
+    agent any
+    environment {
+        package = "atdome"
+        package_version = sh(returnStdout: true, script: "git describe --tags --always --dirty").trim()
+        dockerImageName = "lsstts/conda_package_builder:latest"
+        container_name = "salobj_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
+    }
+
+    stages {
+        stage("Pull Docker Image") {
+            steps {
+                script {
+                sh """
+                docker pull ${dockerImageName}
+                """
+                }
+            }
+        }
+        stage("Start builder"){
+            steps {
+                script {
+                    sh """
+                    docker run --name ${container_name} -di --rm \
+                        --env TS_CONFIG_ATTCS_DIR=/home/saluser/ts_config_attcs \
+                        --env LSST_DDS_DOMAIN=citest \
```

Comment:
I thought the SALOBJ unit tests created a random DDS domain. If we have multiple jobs running, we could potentially have collisions if we peg to specific domain names.

Reply:
salobj creates a random LSST_DDS_DOMAIN on unit tests: https://github.com/lsst-ts/ts_salobj/blob/e303060adb0bdb18906783e0d7ad87a5312302a3/python/lsst/ts/salobj/base_csc_test_case.py#L104

The environment variable just needs to be set.

Reply:
You are correct. I forget I pass one in all the time to containers.